src/main/java/duke/statistic/Counter.java

```diff
@@ -0,0 +1,48 @@
+package duke.statistic;
+
+import duke.command.Command;
+import duke.core.DukeException;
+import duke.storage.CounterStorage;
+
+import java.util.Map;
+
+/**
+ * This is a Counter class that mainly used for counting purpose
+ * which can then to be used for relevant application.
+ *
+ * @author QIAN JIE
+ * @version 1.3
+ */
+public class Counter {
+    private Map<String, Integer> commandTable;
+
+    public Counter(Map<String, Integer> commandTable) {
+        this.commandTable = commandTable;
+    }
+
+    public Map<String, Integer> getCommandTable() {
+        return commandTable;
+    }
+
+    /**
+     * This function is used to run the command counter.
+     *
+     * @param command        the command type that is being processed
+     * @param counterStorage get the counterStorage object
+     * @param counter get the Counter object
+     * @author QIAN JIE
+     * @version 1.3
+     */
+
```

Comment:
Be careful with stray empty lines. Also, with the javadoc, you should use a simple language. "This function.." could simply be "Runs the command counter"

Reply:
fixed. thanks for the suggestion!